src/cntk-model/src/main/python/CNTKModel.py

```diff
@@ -8,6 +8,7 @@
 
 from mmlspark._CNTKModel import _CNTKModel
 from pyspark.ml.common import inherit_doc
+from mmlspark.Utils import *
```

Comment:
Please avoid ``import *``. Also, I'd prefer to keep casing of modules consistent so please consider ``mmlspark.utils`` instead of ``mmlspark.Utils``.